### PR TITLE
Create and add tone of voice guidelines

### PR DESCRIPTION
**Pull request by duffylouise**
Created a new guide for tone of voice for the new brand. Apologies if this is a really long thing to review! Decided not to use the existing style guide as this info feels better as a new guide.

The original doc is here if you need it: https://docs.google.com/document/d/1Cn18K4Z8Ucq-UkzE0gMO6SsWPYAOmh-SisdENXnNamM/edit#

### CODE REVIEW

**Comment by erbridge** [comment]
Needs adding to the navbar, too (in `partials/` somewhere).

File: guides/tone-of-voice.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+* curious - and willing to try new things and improve. As well as helping others to do the same
+* determined - and motivated to tackle difficult things and have high standards
+
+We should aim to show our audience the value of our work, not just tell them about it. That might mean showing things like screenshots or video footage of services, outputs or pictures from user research sessions and so on.
+
+## No need for formality
+
+We work with organisations but we should write for the people who work in those organisations or use their services. That means using the words that real people use.
+
+Think about what you’ve written and whether it’s something you would say out loud. If not, then it’s worth changing your language to sound more approachable - more like you.
```
Comment (erbridge) [inline guides/tone-of-voice.md:33]: I feel called out as a robot :)

Comment (duffylouise) [reply guides/tone-of-voice.md:33]: Ha ha

**Comment by johnwaterworth** [comment]
This looks great.

There is an existing style guide in the Playbook. It has some guidance on specific product and technical terms.
https://playbook.dxw.com/#/guides/style-guide

I suggest adding a section to this guide for specific product and technical terms so we can get rid of the other guide.

**Pull request merged**